Assets/Scripts/Gameplay/Mecha/MechaController.cs

```diff
@@ -72,7 +77,10 @@ public enum View
 
         private Zoom _zoom = Zoom.Default;
         private bool _isRunning;
-        
+
+        private bool isDashing = true;                      // Track if dashing
```

Comment:
You don't really need this boolean as `CurrentMovementMode == MovementMode.Dashing` does the trick

Reply:
Yeah most of the stuff around dashing was to just keep my stuff there in case I am to switch computers for now so that I have a physical copy